internal/node/controlapi.go

```diff
@@ -217,10 +218,22 @@ func handleRun(api *ApiListener) func(m *nats.Msg) {
 				slog.String("vmid", runningVm.vmmID),
 				slog.String("namespace", namespace),
 				slog.String("workload", workloadName),
+				slog.Uint64("workload_size", numBytes),
+				slog.String("workload_sha256", *workloadHash),
 				slog.String("type", *request.WorkloadType),
 			)
 
-		err = api.mgr.DeployWorkload(runningVm, workloadName, namespace, request)
+		req := agentapi.DeployRequest{
+			WorkloadName:    &workloadName,
+			Hash:            *workloadHash,
+			TotalBytes:      int64(numBytes),
+			TriggerSubjects: request.TriggerSubjects,
+			WorkloadType:    request.WorkloadType, // FIXME-- audit all types for string -> *string, and validate...
+			Environment:     request.WorkloadEnvironment,
+			Namespace:       &namespace,
+		}
+
+		err = api.mgr.DeployWorkload(runningVm, request, req)
```

Comment:
I don't love `request` and `req`. Very hard to read. We should be able to consolidate these right?

Reply:
Agree.  See other comment about covering in different PR